scorer_cleanup

```diff
@@ -0,0 +1,33 @@
+#Declare custom scorer
+
+def RMSE(y_true,y_pred, verbose=False):
+    rmse = sqrt(mean_squared_error(y_true, y_pred))
+    if verbose==True:
```

Comment:
you don't need `==True` unless you are super strict; just `if verbose:` is sufficient.

the `else` is also unnecessary.